content/en/scale-agent/concepts/dynamic-accounts.md

```diff
@@ -0,0 +1,139 @@
+---
+title: Dynamic Accounts Architecture and Features
+linkTitle: Dynamic Accounts
+description: >
+  Learn how the Dynamic Accounts feature simplifies migrating accounts from Clouddriver to the Armory Scale Agent in your Armory Continuous Deployment or Spinnaker instance.
+aliases:
+  - /scale-agent/tasks/dynamic-account-options/
+  - /scale-agent/reference/api/dynamic-accounts/
+  - /scale-agent/reference/api/
+---
+
+## Overview of Dynamic Accounts
+
+Rather than [manually configure]({{< ref "scale-agent/reference/config/agent-options#configure-kubernetes-accounts" >}}) each Scale Agent service with the Kubernetes accounts it should manage, you can use the Dynamic Accounts feature to migrate and manage your accounts. Dynamic Accounts provides:
+
+* Manual migration of Clouddriver Kubernetes accounts to the Scale Agent using a REST API
+* Automatic migration of Clouddriver Kubernetes accounts using Clouddriver Account Management
+
+   * Automatic migration requires Armory Continuous Deployment 2.28+ or Spinnaker 1.28+.
+   * Clouddriver Account Management is not enabled by default in Spinnaker or Armory Continuous Deployment. See Spinnaker's [Clouddriver Account Management](https://spinnaker.io/docs/setup/other_config/accounts/) for how to enable this feature in your Spinnaker instance.
+
+### REST API
+
+{{< include "scale-agent/api-overview" >}}
+
+### How to enable and use Dynamic Accounts
+
+First, familiarize yourself with the architecture and features in this guide. Then you can:
+
+1. {{< linkWithTitle "scale-agent/tasks/dynamic-accounts/enable.md" >}}
+1. {{< linkWithTitle "scale-agent/tasks/dynamic-accounts/migrate-accounts.md" >}}
+1. {{< linkWithTitle "scale-agent/tasks/dynamic-accounts/manage-accounts.md" >}}
+
+## Dynamic Accounts glossary
+
+- **Account**: an abstraction of a target cluster or target set of namespaces within a cluster
+- **Credentials source**: any source from which credentials/accounts are read
+- **Endpoint**: the URL segment after the Clouddriver root
+- **Migrate an account**: move a Clouddriver-managed account to the Scale Agent for management
+- **Request**: an instruction that isn’t fulfilled immediately and can have different outcomes; a request can be done through HTTP by the admin or internally by one of the services.
+
+## Architecture
+
+The Scale Agent stores account data in a dedicated table called `clouddriver.kubesvc_accounts`. It does not modify or delete the account data in the `clouddriver.accounts` table.
```

Comment:
"maybe we can change the last part to It does not modify or delete account data in other credential source such as clouddriver.accounts". Because not every customer is using that storage